src/api/instance/instance.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import {type CredrankInput} from "../credrank";
+import {CredGraph} from "../../core/credrank/credGraph";
+import {Ledger} from "../../core/ledger/ledger";
+import {type DependenciesConfig} from "../dependenciesConfig";
+
+/**
+This interface represents a SourceCred Instance, which is simply a read/write
+interface for configurations and outputs of the main SourceCred API.
+ */
+export interface Instance {
+  loadCredrankInput(): Promise<CredrankInput>;
```

Comment:
Naming: might I suggest that the dual to `write` be called `read` rather
than `load`, just for general consistency? (e.g., with `writeFile` and
`readFile` from Node’s `fs` API)

Reply:
accepted